data/private

```diff
@@ -150,10 +152,14 @@ tendawifi.com
 
 # TP-Link router
 tplinkwifi.net
+tplogin.cn
 
 # Windows
 msftconnecttest.com
 msftncsi.com
 
-# 中興路由器
+# Xiaomi router
+miwifi.com
```

Comment:
Use the `full:` prefix whenever you can to avoid mismatch.

Reply:
Same as above.